drivers/vci.c

```diff
@@ -15,6 +15,7 @@
 
 void vci_enable(void)
 {
+#if defined (CONFIG_SOC_FAMILY_MEC)
```

Comment:
instead of if def in this file at multiple places, adding CONFIG_SOC_FAMILY_MEC check in CMakelist to include vci.c in compilation only for MEC family is cleaner way as VCI it self microchip specific feature

Reply:
Hi @VenkatKotakonda thanks for your suggestion ! I found vci.c is no use currently, which instead of corresponding MEC board like 'drivers/vci_mec15xx.c' and 'drivers/vci_mec172x.c'. I removed those of if def in vci.c, but included its in functions deep_sx_enter' and 'deep_sx_exit' from 'app/power_sequencing/deepsx.c'.